src/main/java/nl/workingtalent/bookrental/controller/CopyController.java

```diff
@@ -20,11 +21,24 @@ public class CopyController {
 	private ICopyRepository copyRepo;
 	@Autowired
 	private IBookRepository bookRepo;
+	@Autowired
+	private UserController userController;
 	
+	// TODO change from Copy to status. Does not need to return the created copy.
 	@PostMapping("copy/create/{bookId}")
-	public void createCopy(@RequestBody Copy copy, @PathVariable long bookId) {
+	public Copy createCopy(@RequestHeader(name = "Authorization") String token, @PathVariable long bookId) {
+		
+		// TODO: Change to return status object instead
+		if (!userController.CheckUserPermissions(token)) return null;
+		
 		Book book = bookRepo.findById(bookId).get();
-		copy.setBook(book);
+		Copy copy = new Copy(true, book);
+		
+		book.addCopy(copy);
```

Comment:
waarschijnlijk snap ik de koppeling niet helemaal, 
maar waarom moet je eigenlijk 2x koppelen: je geeft het boek een nieuwe copy & ook de copy wordt aan dat book gekoppeld. Aangezien Spring 1000 dingen onder water doet, kan dit ook niet in 1x?